docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotHashEncoding.cs

```diff
@@ -4,34 +4,35 @@
 using Microsoft.ML.Data;
 using Microsoft.ML.Transforms;
 
-namespace Samples.Dynamic
+namespace Samples.Dynamic.Transforms.Categorical
 {
     public static class OneHotHashEncoding
     {
         public static void Example()
         {
-            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
-            // as well as the source of randomness.
+            // Create a new ML context, for ML.NET operations. It can be used for
+            // exception tracking and logging, as well as the source of randomness.
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable.
-            var samples = new List<DataPoint>()
+            var samples = new List<DataPoint>
```

Comment:
>new List<DataPoint> [](start = 26, length = 19)

why not array? It's also an enumerable and is readonly...

Reply:
good idea.

---
In reply to: [278760256](https://github.com/dotnet/machinelearning/pull/3588#discussion_r278760256) [](ancestors = 278760256)